process_duty_email.py

```diff
@@ -150,87 +143,105 @@ def process_data(self):
-            url_RPKM = get_data(df_RPKM, '"{}"'.format(path_to_RPKM))
-            url_coverage = get_data_GSTT(df_coverage, '"{}"'.format(destination_of_Coverage))
-            url_StG_RPKM = get_data(df_RPKM, '"{}"'.format(StG_transfer_folder + "\\RPKM"))
-            url_StG_coverage = get_data_StG(df_coverage, '"{}"'.format(StG_transfer_folder + "\\coverage"))
-            if len(df_FHPRS.index) >= 1 :
-                destination_of_FHPRS = output_folder + "\\FH_PRS"
-                os.mkdir(destination_of_FHPRS)
-                os.mkdir(StG_transfer_folder + "\\FH_PRS")
-                url_FHPRS = get_data_GSTT(df_FHPRS, '"{}"'.format(destination_of_FHPRS))
-                url_StG_FHPRS = get_data_StG(df_FHPRS, '"{}"'.format(StG_transfer_folder + "\\FH_PRS"))
+            if bool(re.search("NGS\w+_NGS", file_path)): 
+                print("WARNING! More than one NGS project has been found in the file name.\n \
+                    This script currently does not support this operation.\n \
+                    Possible solution: manually split the csv file into separate\n \
+                    projects and rename the files to contain onty one NGS project.\n \
+                    e.g. 002_211119_A01229_0050_BHMYNGDRXY_NGS446_NGS445A_NGS444.csv\n \
+                    to become: 002_211119_A01229_0050_BHMYNGDRXY_NGS446.csv \n \
+                               002_211119_A01229_0050_BHMYNGDRXY_NGS445A.csv \n \
+                               002_211119_A01229_0050_BHMYNGDRXY_NGS444.csv")
+                break
             else:
-                url_FHPRS = ''
-                url_StG_FHPRS = ''
-            all_url =  url_RPKM + url_coverage + url_FHPRS + url_StG_RPKM + url_StG_coverage + url_StG_FHPRS
-            output = download_data(all_url)
-            list_of_stings = file_path.split("\\")
-            save_log_file(output, list_of_stings[-1].replace('.csv','.txt'))
-            print(output)
+                path_to_RPKM = os.path.join(output_folder, "RPKM")
+                destination_of_Coverage = os.path.join(output_folder, "coverage")
+                os.mkdir(path_to_RPKM)
+                os.mkdir(destination_of_Coverage)
+                # import csv file to pandas df
+                csv_file = pandas.read_csv(file_path, index_col=None)
+                # get data frames from the master df
+                df_RPKM = csv_file[csv_file['type'] == 'RPKM']
+                df_coverage = csv_file[csv_file['type'] == 'coverage']
+                df_FHPRS = csv_file[csv_file['type'] == 'FH_PRS']
+                # Create a folder with NGS run name, subfolders cold coverasge and RPKM, download data
+                results = re.search(r"\\(NGS\S+)$", output_folder)
+                foldername = results.group(1)
+                StG_transfer_folder = os.path.join(StG_transfer, foldername)
+                # Make directories in the outgoing folder
+                os.mkdir(StG_transfer_folder)

```

Comment:
should we first test to see if there are any stg samples before doing this? - this can be a future release (please add as issue)